tests/dapr_tests.mk

```diff
@@ -106,11 +107,11 @@ push-perf-app-all: $(PUSH_PERF_APPS_TARGETS)
 
 # start all e2e tests
 test-e2e-all: check-e2e-env
-	DAPR_TEST_NAMESPACE=$(DAPR_TEST_NAMESPACE) DAPR_TEST_TAG=$(DAPR_TEST_TAG) DAPR_TEST_REGISTRY=$(DAPR_TEST_REGISTRY) DAPR_TEST_MINIKUBE_IP=$(MINIKUBE_NODE_IP) go test -p 1 -count=1 -v -tags=e2e ./tests/e2e/...
+	DAPR_TEST_NAMESPACE=$(DAPR_TEST_NAMESPACE) DAPR_TEST_TAG=$(DAPR_TEST_TAG) DAPR_TEST_REGISTRY=$(DAPR_TEST_REGISTRY) DAPR_TEST_MINIKUBE_IP=$(MINIKUBE_NODE_IP) go test -p 2 -count=1 -v -tags=e2e ./tests/e2e/...
```

Comment:
Are we sure the tests can run in parallel?

Reply:
tests can be run in parallel for normal functional tests, but if a test case tests dapr config which apply to the entire test, it won't work. I would keep using single parallel.

Reply:
I did a cursory survey of the test apps, and I did not see any resources shared between them. Also multiple runs have not revealed any race conditions. This is parallelism at the test app level, and not at the individual test level.

Reply:
But, if you have reason to believe otherwise, by all means.

Reply:
Ok, Young Bu and I discussed this. The primary concern is a global config test that will come in the future, but does not yet exist. When we create the global config test, we'll just have to make sure we run it by itself apart from the rest of the test suite. I have added a comment which will reflect this.

Reply:
OK.